nems/util/shr_abort_mod.F90

```diff
@@ -0,0 +1,164 @@
+module shr_abort_mod
+  ! This module defines procedures that can be used to abort the model cleanly in a
```

Comment:
I notice above that you replaced shr_abort_mod with an ESMF abort call - do you still need this?

Reply:
You still need this for for util - since it specifically references cime/src/share/util routines that use shr_kind_mod. We could rewrite those routines - but I did not take that on right now.